addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiAqaraSensorWeatherV1Handler.java

```diff
@@ -0,0 +1,66 @@
+package org.openhab.binding.mihome.handler;
+
+import static org.openhab.binding.mihome.XiaomiGatewayBindingConstants.*;
+
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.thing.Thing;
+
+import com.google.gson.JsonObject;
+
+/**
+ * Handles the Xiaomi Aqara temperature, humidity and pressure sensor
+ *
+ * @author Daniel Walters - Support for Xiaomi Aqara temperature, humidity and pressure sensor
+ */
+public class XiaomiAqaraSensorWeatherV1Handler extends XiaomiSensorBaseHandler {
+
+    private static final String HUMIDITY = "humidity";
+    private static final String TEMPERATURE = "temperature";
+    private static final String VOLTAGE = "voltage";
+    private static final String PRESSURE = "pressure";
+
+    public XiaomiAqaraSensorWeatherV1Handler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    void parseReport(JsonObject data) {
+        parseDefault(data);
+    }
+
+    @Override
+    void parseHeartbeat(JsonObject data) {
+        parseDefault(data);
+    }
+
+    @Override
+    void parseReadAck(JsonObject data) {
+        parseDefault(data);
+    }
+
+    @Override
+    void parseWriteAck(JsonObject data) {
+        parseDefault(data);
+    }
+
+    @Override
+    void parseDefault(JsonObject data) {
```

Comment:
please check if sensor tells the truth in heartbeat messages for temp & hum & pressure & provide debug log